assets/src/edit-story/components/form/switch.js

```diff
@@ -124,12 +124,10 @@ function Switch({ value, disabled, onChange, onLabel, offLabel }) {
     [onChange]
   );
   const ref = useRef();
-  useKeyDownEffect(
-    ref,
-    ['space', 'enter', 'left', 'right'],
-    () => handleChange(!value),
-    [handleChange, value]
-  );
+  useKeyDownEffect(ref, ['left', 'right'], () => handleChange(!value), [
```

Comment:
This now wouldn't change the value on `Enter`/`Space`, is that intended?

Reply:
<kbd>space</kbd> already worked even without this handler as it was default behavior. After this fix, <kbd>enter</kbd> also works by browser default behavior. This is according to my tests at least, are you experiencing otherwise?

Reply:
Correct, neither of these seem to work when testing in Chrome.

Reply:
You're right, reverted in 78a23aec4.